addons/medical_status/CfgActions.hpp

```diff
@@ -0,0 +1,9 @@
+class CfgActions {
+    class None;
+    class Gear: None {
+        show = 0;
+    };
+    class OpenBag: None {
```

Comment:
less commonly used but there also is an action to open inventory for backpacks on the ground